src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/person/domain/OppfolgingstilfelleDag.kt

```diff
@@ -140,6 +140,7 @@ fun OppfolgingstilfelleDag.isArbeidsdag() =
                     or (Tag.SYKEPENGESOKNAD and Tag.ARBEID_GJENNOPPTATT)
                     or (Tag.SYKEPENGESOKNAD and Tag.BEHANDLINGSDAGER)
                     or (Tag.SYKMELDING and Tag.BEHANDLINGSDAGER)
+                    or (Tag.SYKMELDING and Tag.REISETILSKUDD)
```

Comment:
Denne tag-kombinasjonen må også ligge i lista over prioriteringer siden det ofte vil være flere tag-kombinasjoner som gjelder for en dag og da må koden vite hvilken av dem som "vinner".